tracim/tracim/model/data.py

```diff
@@ -435,48 +439,516 @@ def toDict(self):
                     label=self.label,
                     priority=self.priority)
 
-class Content(DeclarativeBase):
```

Comment:
The diff is ugly here ... ``Content`` class code available here: https://github.com/buxx/tracim/blob/dev/content/tracim/tracim/model/data.py#L601